tests/communication/ros2/test_connectors.py

```
@@ -0,0 +1,193 @@
+
+from .helpers import ActionServer_ as ActionServer
+from .helpers import (
+    MessagePublisher,
+    MessageReceiver,
+    ServiceServer,
+    multi_threaded_spinner,
+    ros_setup,
+    shutdown_executors_and_threads,
+)
+
+ros_setup  # type: ignore prevents pytest from complaining about unused fixture
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)